### PR TITLE
Moved blame subscriber.

### PR DESCRIPTION
**Pull request by dantleech**
This PR moves the blame subscriber from the document manager to sulu for reasons explained in https://github.com/sulu-io/sulu-document-manager/issues/26

This PR does not depend on any other PR and can be merged whenever if the tests are green.

### CODE REVIEW

**Comment by dantleech** [comment]
@wachterjohannes this also includes the `tokenStorage=null` fix.

**Comment by dantleech** [comment]
TODO: Need to make the documents implement this interface.

**Comment by wachterjohannes** [comment]
@dantleech i have added this fix into this PR https://github.com/sulu-io/sulu/pull/1355 but i think its no problem to merge (:

**Comment by dantleech** [comment]
Updated. If green it is OK for me.

**Comment by danrot** [comment]
Why is this necessary? It is a bit problematic, because it is a BC break... If we really need it (what I TBH doubt), there should at least be a UPGRADE note.

**Comment by dantleech** [comment]
On Mon, Jul 13, 2015 at 12:45:53AM -0700, Daniel Rotter wrote:
>    Why is this necessary? It is a bit problematic, because it is a BC
>    break... If we really need it (what I TBH doubt), there should at least be
>    a UPGRADE note.

The BlameSubscriber depends on the Sulu UserInterface and the Symfony
TokenStorage class, it has never belonged in the DocumentManager (and
indeed the tests for it were skipped).

I will add to the UPDATE note.

**Comment by chirimoya** [comment]
ping @dantleech

**Comment by dantleech** [comment]
Updated, needed to update to use Metadata instead of the property encoder. And added UPGRADE note.
Lets see what the tests say.

**Comment by dantleech** [comment]
Tests green.

File: src/Sulu/Component/Content/Document/Behavior/BlameBehavior.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Document\Behavior;
+
+interface BlameBehavior
```
Comment (danrot) [inline src/Sulu/Component/Content/Document/Behavior/BlameBehavior.php]: Can you add an explanation here?

**Comment by webmozart** [comment]
Replaced by #1983.

**Pull request closed without merging**